cmd/clock_test.go

```diff
@@ -0,0 +1,32 @@
+// +build test
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+// Clock functions similarly to clock.Default(), but the returned value can be
+// changed using the FAKECLOCK environment variable in test environments.
+//
+// The FAKECLOCK env var is in the time.UnixDate format, returned by `date -d`.
+func Clock() clock.Clock {
+	if tgt := os.Getenv("FAKECLOCK"); tgt != "" {
+		targetTime, err := time.Parse(tgt, time.UnixDate)
+		if err != nil {
```

Comment:
(Should've caught this in the last round sorry) you should use a `FailOnError` here since this is essentially replicating what that method does.